website/content/community/release-process.md

```diff
@@ -7,12 +7,25 @@ This is the process to follow to make a new release. This should
 really be automated someday, but releases are juuuust infrequent
 enough that it's less annoying to just do the work each time.
 
+
+
 ### Check the issue tracker milestone
 
 Are you actually ready to release? Check the milestone on github and
 verify that all its issues are closed. If there are open issues,
 you'll have to either resolve them, or bump to the next version.
 
+
+### ### Cherry-pick relevent commits
+
+MetalLB uses release branches to track releases. Relevant commits should be cherry-picked onto the release branch.
+For example,
+```
```

Comment:
I'd add an empty line before the snippet.

Reply:
A new line has been added, marking as solved